src/core_plugins/metrics/public/components/aggs/math.js

```diff
@@ -56,19 +56,21 @@ class MathAgg extends Component {
               />
             </div>
             <div className="vis_editor__row_item">
-              <div className="vis_editor__label">
-                Basic math (see <a href="http://mathjs.org/docs/expressions/syntax.html" target="_blank">MathJS</a>) - Variables
-                are keys on the <code>params</code> object, i.e. <code>params.&lt;name&gt;</code> To access all the data use
-                <code>params._all.&lt;name&gt;.values</code> for an array of the values and <code>all.&lt;name&gt;.timestamps</code>
-                for an array of the timestamps.  <code>params._timestamp</code> is available for the current bucket&apos;s timestamp
-                and <code>params._index</code> is available for the current bucket&apos;s index.
-              </div>
+              <label className="vis_editor__label" aria-label="script">Expression</label>
```

Comment:
You either need to use either:

```html
<label for="mathExpressionInput">...</label>
<input id="mathExpressionInput">
```

or

```html
<label id="mathExpressionLabel">...</label>
<input aria-labelledby="mathExpressionLabel">
```

That way, you gave the label a new name that is read out by a screenreader (so it reads out "script" instead of "Expression") and the input field is referenced by an element, with the id "script" (that most likely doesn't exist anywhere in the page :D)

From the two solutions, if possible favor the first, since it will also add the additional usability to every user, by automatically jumping to the input field, when a user clicks the label. The second should be used, when you have multiple elements, in a list, that are labeled by the same label (e.g. our From/To range inputs).

Reply:
Could you also give the `div` with the longer explanation an `id` and reference that `id` via an `aria-describedby` on the `input` field? That way if you pause on the field, after it read outs it's label "Expression" it will also give you the longer explanation as a description (which is usually meant to provide more information about an element).